src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.utils.smartshuffleboard.SmartShuffleboard;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class IntakeSubsystem extends SubsystemBase{
+    private final WPI_TalonSRX intakeMotor1;
+    private final WPI_TalonSRX intakeMotor2;
+
+    public IntakeSubsystem() {
+        this.intakeMotor1 = new WPI_TalonSRX(Constants.INTAKE_MOTOR_1_ID);
+        this.intakeMotor2 = new WPI_TalonSRX(Constants.INTAKE_MOTOR_2_ID);
+        
+        intakeMotor1.setInverted(false);
+        intakeMotor2.setInverted(false);
+
+        //Configure current limits *VALUES HAVE TO BE TWEAKED*
+        intakeMotor1.configPeakCurrentLimit(Constants.INTAKE_MOTOR_PEAK_CURRENT_LIMIT);
+        intakeMotor1.configPeakCurrentDuration(Constants.INTAKE_MOTOR_PEAK_CURRENT_DURATION);
+        intakeMotor1.configContinuousCurrentLimit(Constants.INTAKE_MOTOR_CONTINUOUS_CURRENT_LIMIT);
+        intakeMotor1.enableCurrentLimit(true);
```

Comment:
Add a constant for whether the current limit should be enabled, and start with it false. We'll have to find the right numbers and if we run with current limiting to begin with it will give us incorrect behavior.